docs/60_PACKAGE.md

```diff
@@ -51,6 +51,35 @@ directory.  Defaults to the docroot.
 packages directory. This can be overridden by calling grunt package with the
 `--name` parameter.
 
+**archive**: Used to enable the `compress` option via the configuration instead
+of the command line. Defaults to false.
+
+**rsync**: If set to true, rsync will be used instead of file copying for
+improved performance. This automatically enables the `compress` option and will
+write the archive to the package destination.
+
+**vmData**: This path is used as the intermediate generation directory for the
+package when using the `rsync` option. By selecting a volume mounted in your
+VM `/data` area, this can significantly improve performance. The archive is
+still moved to the normal package destination when complete.  Defaults to
+`build/vm_data`.
```

Comment:
This is the super important part of this PR.  Without the virtual intermediate directory there was no way to get anything performant and people are bypassing gdt on their projects because of this.  I know GDT is not "targetting" at containers, which is why this option isn't enabled by default.

The idea here is that in your docker container you mount a volume that gdt can use for various purposes to improve performance.  Only tackling the packaging right now, but I can imagine other commands using this also.  So don't want it to be within "build/package".  The /data is just an example that pertains to our own container, but that isn't anywhere in the GDT except the docs as an example.  Since our docs already reference many P2 practices, I don't see this as a problem right now (see the CI docs for example).

Still, specific updates to this doc file are welcome if you want to wordsmith it a bit.